addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1335,15 +1359,25 @@ var SnippetsMenu = Widget.extend({
                     $els.attr('data-name', name).data('name', name);
                 }
 
+                const isSanitizeForbidden = $snippet.data('oeForbidSanitize');
+                const isSanitizeForbiddenText = isSanitizeForbidden ?
+                    _t("This snippet cannot be dropped in sanitized model fields for security reasons") : "";
```

Comment:
Don't forget you need to update the .pot file for new translatable terms in stable: https://github.com/odoo/odoo/wiki/Translations#adding-new-terms-when-suggesting-fixes-in-stable-versions

But is that sentence ok? I don't think the notion of "sanitized", "model" and "fields" are known by users... and I don't like saying "for security reasons" to users either cc @rdeodoo @pierre-pde

Reply:
Agreed, it probably should just say something along the lines of the snippet being incompatible with the field.

Reply:
Maybe, although strange to see this message and still be able to drag & drop in some places on this "object" page. Also, I think @bso-odoo shows that tooltip whatever page you are on? Either need some more code to only show this tooltip on pages with sanitized fields or maybe use a sentence like this one: "For technical reasons, this block may be incompatible with some areas on this page".

Reply:
EDIT: actually, that tooltip is just totally useless no? Either you can drop the snippet and it is confusing, either you can't and there is a message on the dropzones where you can't. And the message in those dropzone s("For technical reasons, that snippet cannot be dropped in such an area", the message I suggested before) could be better too, maybe: "For technical reasons, this block cannot be dropped here". "Snippet" is not a word to show to users either. (Reminder: @rdeodoo suggested we show those dropzone anyway so that the user does not think it is a bug that some dropzones appear (unsanitized fields) while some does not (sanitized fields).